src/setup/MongoDB.ts

```diff
@@ -71,7 +64,7 @@ export async function askForMongoDBUrl(): Promise<string> {
       type: "input",
       name: "url",
       message: "Enter your MongoDB URL:",
-      default: process.env.MONGO_DB_URL,
+      default: "mongodb://localhost:27017",
```

Comment:
1. What happens if the parameter is already defined and it's not `localhost` or port `27017`? Will the application break?
2. Can a ternary be used?

Reply:
If we have already defined then it's going to show us
<img width="406" alt="image" src="https://github.com/PalisadoesFoundation/talawa-api/assets/76090263/7fa3afdc-c1e1-4a5e-a54c-e9174ec4e673">

Reply:
but if another url is set in our env, it won't show that, it'll always show "mongodb://localhost:27017"
the line should be this `default: process.env.MONGO_DB_URL ?? "mongodb://localhost:27017"`

Reply:
check the screenshot in previous comment, it's showing different. The port is different from default.

default means - when we have Yes/No type, do you see one of them marked automatically and u press enter?

Reply:
the screenshot is for a different check

Reply:
You said it doesn't show from env. That url is from env itself, check the port. It's different port from default

Reply:
what? we're talking about the line that Peter commented on, you've hard coded `mongodb://localhost:27017` there, so when that function runs, it'll always show `mongodb://localhost:27017` for that prompt.

> You said it doesn't show from env. That url is from env itself, check the port. It's different port from default

the screenshot you've provided is just a log, that logs [link](https://github.com/PalisadoesFoundation/talawa-api/blob/7bd0b3f5c5fab60d705567a5a06a3b2f57006278/setup.ts#L814), it will show what's in the `.env`